sdmetrics/single_table/new_row_synthesis.py

```diff
@@ -148,8 +148,8 @@ def compute_breakdown(
             except TypeError:
                 if len(real_data) > 10000:
                     warnings.warn(
-                        'Unable to optimize query. For better formance, set the '
-                        '`synthetic_sample_size` parameter or upgrade to Python 3.8'
+                        'Unable to optimize query. For better performance, set the '
+                        '`synthetic_sample_size` parameter or upgrade to Python 3.9 or higher'
```

Comment:
is this still necessary? If this library isn't supported on python <3.9, why would they need to upgrade?